code-examples/php/usage-examples/find.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require_once __DIR__ . "/vendor/autoload.php";
+
+$uri = "mongodb://localhost:27017";
+
+$client = new MongoDB\Client($uri);
+
+try {
+    $collection = $client->sample_mflix->movies;
+
+    $filter = ['runtime' => ['$lt' => 15]];
+    $options = ['sort' => ['title' => 1]];
```

Comment:
I'll note that in the other files you provided arguments directly to the method. If you want to construct them in advance so the variable names can self-document the code, that's fine, but be consistent.